tests/core/tools/manual/escapeCss.md

```diff
@@ -0,0 +1,11 @@
+@bender-tags: 4.16.1, bug, 681
```

Comment:
Please lowercase file name. I know that camel case convention is useful, but there are edge cases where our test runner can have problem with such files. Because of that, its more like informal convention, that all test files are named lower cased.